code/modules/projectiles/gun.dm

```diff
@@ -187,7 +188,9 @@
 			return FALSE
 	if(twohanded)
 		if(!wielded)
-			to_chat(user, SPAN_DANGER("The gun is too heavy to shoot in one hand"))
+			if (world.time >= recentwield + 10)
```

Comment:
Change 10 to 1 SECONDS

Easier to read

Reply:
Done